### PR TITLE
Specify acceptable format for CLI-specific bool envvars

### PR DESCRIPTION
**Pull request by soulshake**
(`CONVOX_WAIT`, `CONVOX_DEBUG`, `RACK_PRIVATE`)

## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by soulshake** [comment]
[Document which regions EFS is available in.](https://trello.com/c/3CeHlrUm/2072-document-which-regions-efs-is-available-in)

**Comment by soulshake** [comment]
[Check docs for yes/no CLI env vars --> true/false](https://trello.com/c/DsliJWLB/2049-check-docs-for-yes-no-cli-env-vars-true-false)

File: _reference/cli-environment-variables.md [hunk 1]
```diff
@@ -66,6 +72,9 @@ Instance type for a dedicated build cluster to create when running `convox insta
 
 Make Racks private by default when running `convox install` (overridden by `--private`).
 
+| Default behavior      | `false`                    |
+| Supported values      | `1`, `true` / `0`, `false` |
```
Comment (ianmalott) [inline _reference/cli-environment-variables.md]: Thoughts on keeping these "Supported values" lists fully comma-separated? Seeing the `/` gave me pause. Might just be that it's late in the day :)

Comment (soulshake) [inline _reference/cli-environment-variables.md]: Good point, fixed

**Pull request merged**